docs(query-logs): hint for adding a fulltext option to the searching column (#1070)
GreptimeTeam/docs: docs/nightly/en/user-guide/logs/query-logs.md

> - `column_name`: The column to perform the full-text search on, which should contain textual data of type `String`.
> - `search_query`: A string containing query statement which you want to search for. See the [Query Statements](#query-statements) section below for more details.
> 
> :::tip
> It is strongly recommended that the `column_name` column contains a full-text index to ensure efficient search operations. See the [Full-Text Index for Accelerated Search](#full-text-index-for-accelerated-search) section for more details.
> :::

Reviewer: ```suggestion
- `column_name`: The column to perform the full-text search on, which should contain textual data of type `String`. The [full-text index](#full-text-index-for-accelerated-search) must be created on this column to optimize queries.
- `search_query`: A string containing query statement which you want to search for. See the [Query Statements](#query-statements) section below for more details.
```
